data_steward/analytics/cdr_ops/vocabulary_upload_qc.py

```diff
@@ -51,17 +59,24 @@
                    vocabulary_dataset_new=vocabulary_dataset_new)
 execute(client, query, max_rows=True)
 
-# # Removed PPI. Where ppi concepts are not present in the new vocabulary
+#
+# # PPI concept comparison, retired or removed concepts 
+#
+# > - The dataframe will contain all PPI concepts that were newly deprecated or removed from Athena. <br>
+# > - Generally none would be deprecated or removed. <br>
+# > - If the dataframe is not empty contact the survey team to inquire about the concepts.<br>
```

Comment:
I like this being specific to the PPI vocabulary.  This comment is related to a future scope issue we can create.  

I would like to do something similar with non-PPI concepts that are used to generalize or suppress information.  This could help us loop in privacy earlier when a non-PPI concept is deprecated and a potentially new concept is introduced.  Would you like to create a jira issue for this?